src/client/packaging/npm/package.json

```
@@ -28,6 +28,6 @@
   },
   "devDependencies": {
     "jest": "^27",
-    "shx": "^0.3.3"
+    "shx": "^0.3.4"
```

bad robot, no biscuit
```suggestion
    "shx": "^0.3.3"
```